### DISCUSSION TITLE
Integration Issue

### DISCUSSION THREAD

**Discussion by zlakes01** [wills106/homeassistant-solax-modbus#494] [General]
I have found this message in the Core Logg:
"Please report issue to the custom integration author for solax_modbus doing blocking calls at custom_components/solax_modbus/__init__.py, line 342: return self._client.read_input_registers(address, count, **kwargs)"
Should I report it as BUG?

**Comment by wills106** [comment]
This is covered in the [Readme](https://github.com/wills106/homeassistant-solax-modbus#known-issues) and #379

**Reply by zlakes01** [reply]
Thanks a lot.
